lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -2,36 +2,6 @@
   position: relative;
 }
 
-.container--campaign-gallery {
-  background: $yellow;
-
-  .gallery--reportback > li {
-    width: 100% !important;
-    margin-bottom: $base-spacing;
-
-    @include media($medium) {
-      width: 50% !important;
-      margin-bottom: 0;
-    }
-  }
-
-  .gallery--extended > li {
-    width: 100% !important;
-
-    @include media($medium) {
-      width: 50% !important;
```

Comment:
Hah, yeah! This was from the ill-fated "Gallery" page from the original prototype. Definitely wasn't ever meant to be production ready code. 😉